### DISCUSSION TITLE
No response from GitHub Support

### DISCUSSION THREAD

**Discussion by hrs113355** [community/community#54211] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Our organization currently has over 35 paid users, and we've encountered a billing issue. We tried to open a support ticket to inquire about payment details 13 days ago, but we haven't received any response yet.

Has anyone else faced similar issues, and if so, how did you manage to resolve them? Any advice on the appropriate course of action or getting a response from GitHub Support would be greatly appreciated.

**Comment by robert-osborne** [comment]
We have a billing issue caused by our corporate address changing.  We are now locked out of paid services and are not getting any response to our support ticket.  Did you resolve this?
